packages/commerce-plugin-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/commerce-plugin-tools",
-  "version": "0.2.1-2",
+  "version": "0.2.1-3",
```

Comment:
This is a breaking change of the public API, we could follow Semver and bump the leftmost nonzero digit in the version string.